### PR TITLE
AMQP-416: Deprecate id on <listener-container/>

### PR DESCRIPTION
**Pull request by garyrussell**
JIRA: https://jira.spring.io/browse/AMQP-416

Use `<listener/>` id attribute alone.

### CODE REVIEW

File: src/reference/asciidoc/whats-new.adoc [hunk 1]
```diff
@@ -46,6 +46,13 @@ See <<request-reply>> for more information.
 (Also available in _1.4.4_ as setter on the `RabbitTemplate`).
 
 
+===== Listener Container Bean Names (XML)
+
+The `id` attribute on the `<listener-container/>` element is deprecated and ignored.
+Starting with this release, the `id` on the `<listener/>` child element us used alone to name the listener container bean.
+Normal Spring bean name overrides are applied; if a later `<listener/>` is parsed with the same `id` as an existing beam, the new definition will override the existing one.
```
Comment (artembilan) [inline src/reference/asciidoc/whats-new.adoc]: typo: s/beam/bean

File: src/reference/asciidoc/whats-new.adoc [hunk 2]
```diff
@@ -46,6 +46,13 @@ See <<request-reply>> for more information.
 (Also available in _1.4.4_ as setter on the `RabbitTemplate`).
 
 
+===== Listener Container Bean Names (XML)
+
+The `id` attribute on the `<listener-container/>` element is deprecated and ignored.
+Starting with this release, the `id` on the `<listener/>` child element us used alone to name the listener container bean.
```
Comment (artembilan) [inline src/reference/asciidoc/whats-new.adoc]: typo: s/us/is

**Comment by artembilan** [comment]
Well, and here you added `What's New`.
So, what is your rule to do that or not? :smile:

**Comment by artembilan** [comment]
Merging, anyway, with those typos fixes

File: src/reference/asciidoc/whats-new.adoc (hunk 1 above)
Comment (artembilan) [inline src/reference/asciidoc/whats-new.adoc]: And one more mote to the docs.
Looks like you edit them outside of IDE, so `120 line size` rule is missed.
It is so easy to review locally, plus my changes break lines.
WDYT ?

Comment (artembilan) [inline src/reference/asciidoc/whats-new.adoc]: BTW, can we somehow to configure Travis to build `reference` as well?
This your is OK, but we really sometimes miss to check the docs locally...

**Comment by artembilan** [comment]
Merged as https://github.com/spring-projects/spring-amqp/commit/822fe48c963617a8e6157542be1059b43d909d2e

File: src/reference/asciidoc/whats-new.adoc (hunk 1 above)
Comment (garyrussell) [inline src/reference/asciidoc/whats-new.adoc]: Hmmm - I haven't been applying the 120 char rule to asciidoc - only new line per sentence. I suppose we should use the 120 char rule too.

I will look at travis.

**Pull request closed without merging**